Source/Core/DolphinQt2/Config/Mapping/Hotkey3D.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Dolphin Emulator Project
```

Comment:
Yeahhh I feel you, an architectural tweak could probably do it a lot more effectively than in just this one instance. I'm down to push that off to another PR.